edb/lib/std/30-jsonfuncs.edgeql

```diff
@@ -74,6 +74,16 @@ std::json_get(
     $$;
 };
 
+CREATE FUNCTION
+std::json_opt(json: OPTIONAL std::json) -> std::json
```

Comment:
Hm, not sure this needs to be a public function.  Why not just use `?? <json>'null'`?

Reply:
It seemed like a useful enough function, and I was getting squeamish about the amount of code we were generating, but it's probably fine to just do `??`

Reply:
scalar `??` shouldn't be too bad?

Reply:
Yeah, I'm also against adding sugar functions. Let's get rid of it.

Reply:
In general, adding a new stdlib function is a big deal. Let's not add them in some PRs hidden amongst bigger changes. We should have a separate PR/review process for new APIs.